packages-ui/roosterjs-react/lib/ribbon/type/RibbonButton.ts

```diff
@@ -22,6 +22,12 @@ export default interface RibbonButton<T extends string> {
      */
     flipWhenRtl?: boolean;
 
+    /**
+     * Set to false if we need to enable insert link button on KeyDown event.
+     * @default True insert link button on KeyDown event is disabled by default.
+     */
+    disableInsertLink?: boolean;
```

Comment:
RibbonButton is a common interface. We should not add a property that is only used by one single button. When people create new button they will be confused why I need to care something related to "insertlink"